powershell/build.ps1

```diff
@@ -11,6 +11,7 @@ if ($LASTEXITCODE -ne 0) {
 
 $powershellFolder = Join-Path -Path $cliFolder -ChildPath "powershell"
 Copy-Item -Path "$PSScriptRoot/codeql-extractor.yml" -Destination $powershellFolder -Force
+Copy-Item -Path "$PSScriptRoot/downgrades" -Destination $powershellFolder -Recurse -Force
```

Comment:
Why are we removing the downgrades folder from the codeql-cli/powershell folder? Don't the upgrades and downgrades need to be there?

Reply:
This line copies the downgrades folder from [this folder](https://github.com/microsoft/codeql/tree/main/powershell) to the codeql-cli/powershell folder (just like we do for other files and folders in this script when we inject the PowerShell extractor into the CLI folder)

Reply:
Oh wow I must have reviewed this pre-coffee... I thought the line was removed for some reason, not added. Yes of course this is great